ioos_catalog/tasks/harvest.py

```diff
@@ -700,6 +716,69 @@ def erddap_geojson_url(self, coord_names):
         url_res.close()
         return gj
 
+
+    @classmethod
+    def get_time_from_dim(cls, time_var):
+        """Get min/max from a NetCDF time variable and convert to datetime"""
+        ndim = len(time_var.shape)
+        if ndim == 0:
+            ret_val = time_var.item()
+            res = ret_val, ret_val
+        elif ndim == 1:
+            # NetCDF Users' Guide states that when time is a coordinate variable,
+            # it should be monotonically increasing or decreasing with no
+            # repeated variables. Therefore, first and last elements for a
+            # vector should correspond to start and end time or end and start
+            # time respectively. See Section 2.3.1 of the NUG
+            res = time_var[0], time_var[-1]
+        else:
+            # TODO: use more efficient algorithm for finding max here
+            try:
+                if hasattr(time_var, 'calendar'):
+                    times = num2date(var[:], var.units, var.calendar)
+                else:
+                    times = num2date(var[:], var.units)
+                return np.min(times), np.max(times)
```

Comment:
Could you change this, just set the time to an unknown value. If you try to access the variable, it could crash the service or produce an out of memory error, which, coincidentally will crash the service.